include/SFML/Window/ContextSettings.hpp

```diff
@@ -76,6 +81,8 @@ struct ContextSettings
     unsigned int majorVersion;      ///< Major number of the context version to create
     unsigned int minorVersion;      ///< Minor number of the context version to create
     Uint32       attributeFlags;    ///< The attribute flags to create the context with
+    std::string  vendorName;        ///< Name of the vendor of the OpenGL implementation
+    std::string  rendererName;      ///< Name of the OpenGL renderer implementation
```

Comment:
Although this is an obvious place for adding these members, it's strange because you can set them but this would have no effect, they are just relevant when you read the context settings back from the window.